openlibrary/core/processors/readableurls.py

```diff
@@ -175,9 +176,6 @@ def match(path):
     if thing and thing.type.key == _type:
         title = thing.get(_property) or default_title
         try:
-            # Explicitly only run for python3 to solve #4033
-            from urllib.parse import quote_plus
-
             middle = '/' + quote_plus(h.urlsafe(title.strip()))
         except ImportError:
```

Comment:
the try / except block here is not longer needed, I think this was for when python2 and 3 were supported.